index.d.ts

```diff
@@ -209,6 +209,7 @@ declare module 'warframe-items' {
         heavySlamRadialDamage?: number;
         heavySlamRadius?: number;
         windUp?: number;
+        attacks: (Attack|SlamAttack)[];
```

Comment:
The majority of items components doesn't have attacks.

Suggestion:
`attacks?: (Attack|SlamAttack)[];`

Reply:
It'll still be an empty array, which would always exist

Reply:
If so then there's a fix to be made because not every component have an empty attacks array.

Reply:
yeah, i ended up changing it, but yeah, I see your point now, since this is on the Item interface, not a Weapon interface